src/Main/TextUserInterface.cpp

```diff
@@ -1515,6 +1536,7 @@ namespace VeraCrypt
 						kdf,
 						false,
 						options.Keyfiles,
+                        true,
```

Comment:
Same as above: EMVOption parameter is always set to true regardless of the actual EMV option value.